### PR TITLE
As a search user I want to be able to order results by date, rather than relevance/score

### PR DESCRIPTION
**Pull request by Sandravaphilips**
resolves https://github.com/laws-africa/peachjam/issues/519
image

### CODE REVIEW

File: peachjam/js/components/FindDocuments/index.vue [hunk 1]
```diff
@@ -66,8 +66,25 @@
...
+              <div class="mb-3 sort-body">
                 <div>{{ searchInfo.count }} documents found.</div>
+                <div class="sort__inner">
+                  Sort by
+                  <select
+                    v-model="ordering"
+                    class="ms-2"
+                  >
+                    <option value="-score">
+                      Revelance
```
Comment (longhotsummer) [inline peachjam/js/components/FindDocuments/index.vue:78]: Relevance

**Review by longhotsummer — CHANGES_REQUESTED** [review]

**Comment by longhotsummer** [comment]
Musango can assist with ensuring the strings will be tagged for translation

File: peachjam/js/components/FindDocuments/index.vue (hunk 1 above)
Comment (musangowope) [reply peachjam/js/components/FindDocuments/index.vue]: covered in this pr https://github.com/laws-africa/peachjam/pull/535

**Review by musangowope — APPROVED** [review]

**Review by musangowope — APPROVED** [review]

File: peachjam/js/components/ListFacets.vue [hunk 2]
```diff
@@ -8,6 +8,15 @@
       :loading="loading"
     />
   </form>
+  <!-- DOM Hack for i18next to parse facet to locale json. i18next skips t functions in script element -->
+  <div v-if="false">
+    {{ $t('Regional Body') }}
```
Comment (longhotsummer) [inline peachjam/js/components/ListFacets.vue:13]: #427 would be a good way to get this handled on the server.

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**